integrations/timescaledb/timescaledb-metrics/sensu-integration.yaml

```
@@ -0,0 +1,52 @@
+---
+api_version: catalog/v1
+type: Integration
+metadata:
+  namespace: timescaledb
+  name: timescaledb-metrics
+spec:
+  class: supported
+  provider: metrics
+  display_name: TimescaleDB Metrics
+  short_description: Send Sensu Metric Data to TimescaleDB
+  supported_platforms:
+    - linux
+  tags:
+    - timescaledb
+  contributors:
+    - "@sensu"
+  prompts:
+    - type: section
+      title: Secrets Management
+    - type: markdown
+      body: |
+        The TimescaleDB Metrics integration requires a secret DSN for connecting to TimescaleDB.
```

```suggestion
        The TimescaleDB Metrics integration requires a [Connection URI] (sometimes referred to as a "Data Source Name", or DSN) for connecting to TimescaleDB.
        
        [Connection URI]: https://www.postgresql.org/docs/current/libpq-connect.html#id-1.7.3.8.3.6
```

I couldn't find a single reference to "DSN" in the Postgres docs anywhere. Weird!

---

Glad you pointed that out! Here's an issue https://github.com/sensu/sensu-timescaledb-handler/issues/9

tl;dr What [postgres docs](https://www.postgresql.org/docs/14/libpq-connect.html#LIBPQ-CONNSTRING) call Connection Strings take two forms, key/value and connection URIs. I (for reasons that elude me) have always associated "DSN "with the key/value style, and "Connection URI" with the URI style.